extension/src/popup/styles/global.scss

```diff
@@ -113,8 +113,3 @@ a {
 .sds-theme-dark ol {
   color: inherit !important;
 }
-
-// SDS overrides
-.Heading--lg {
-  line-height: normal;
```

Comment:
I double checked that this override was only being used in one place so removing this global override should not have any unexpected results